Webtest.sh

```diff
@@ -21,8 +21,10 @@ CURL="docker exec fortio_server /usr/local/bin/fortio load -curl -loglevel $LOGL
 $CURL https://istio.io/robots.txt
 # Check that browse doesn't 404s
 $CURL ${BASE_FORTIO}browse
-# Check we can connect, and run a QPS test against ourselves through fetch
+# Check we can connect, and run a http QPS test against ourselves through fetch
 $CURL "${BASE_FORTIO}fetch/localhost:8080$FORTIO_UI_PREFIX?url=http://localhost:8080/debug&load=Start&qps=-1&json=on" | grep ActualQPS
+# Check we can connect, and run a grpc QPS test against ourselves through fetch
+$CURL "${BASE_FORTIO}fetch/localhost:8080$FORTIO_UI_PREFIX?url=localhost:8079&load=Start&qps=-1&json=on&runner=grpc" | grep ActualQPS
```

Comment:
@danehans you raised a good point about the intended usage of grpc mode. I added an acceptance test here. let me know if this helps

Reply:
maybe grep for something GRPC specific like `Health SERVING`

Reply:
the grep is on the result json object which is currently shared between the http/grpc modes. the result object does not have anything grpc-specific for now.
`Health SERVING` is printed to stdout. not to the json file. correct me if I understood this wrong.

Reply:
k can you file a follow up that the GRPC json is missing the equivalent of the Http code map that http has
(don't need to fix everything in this 1 pr if you don't have time)

Reply:
not sure that I understand this. the grpc object does have a retcode map object eg from a sample run of 3000 queries on `localhost:8079`
```
  "RetCodes": {
    "1": 3000
  }
```

however, the json doesn't contain a grpc-specific message eg `Health SERVING`, this is logged to stdout, hence can't be grep-ed for in the json result.

are you suggesting that the result.json should contain http/grpc specific messages?

Reply:
so the code printing stdout is
		fmt.Printf("Health %s : %d\n", k.String(), total.RetCodes[k])

for some reason the Json has the numerical value "1" instead of "SERVING" - I wonder if that's fixable easily with some annotation on the struct

but we can postpone - I guess just grep for "1": 100 and use exactly 100 (-n 100)

Reply:
ah, now i get it! will give it a shot

Reply:
#161

Reply:
not sure how the automation works on this repo? will referencing the issue auto-close it if PR is merged?